lib/bolt/puppetdb/config.rb

```diff
@@ -10,12 +10,16 @@ class Config
         DEFAULT_TOKEN = File.expand_path('~/.puppetlabs/token')
         DEFAULT_CONFIG = { user: File.expand_path('~/.puppetlabs/client-tools/puppetdb.conf'),
                            global: '/etc/puppetlabs/client-tools/puppetdb.conf',
-                           win_global: 'C:/ProgramData/PuppetLabs/client-tools/puppetdb.conf' }.freeze
+                           win_global: File.expand_path(
+                             '%CSIDL_COMMON_APPDATA%/PuppetLabs/client-tools/puppetdb.conf'
```

Comment:
With the win32-dir gem, we can use `File.join(Dir::COMMON_APPDATA, 'PuppetLabs/client-tools/puppetdb.conf')`